### PR TITLE
feat: add views for whats-new feature

### PR DESCRIPTION
**Pull request by siddarth2824**
## Problem

Add views for "What's New" feature

Closes [#3987]

## Solution

**Breaking Changes** 

- [X] No - this PR is backwards compatible  

## Before & After Screenshots
New stories added as `WhatsNew` under `Feature`

### CODE REVIEW

File: frontend/src/hooks/useMdComponents.tsx [hunk 1]
```diff
@@ -44,6 +50,7 @@ export const useMdComponents = ({
           {...textStyles}
         />
       ),
+      ul: ({ node, ...props }) => <UnorderedList {...props} {...textStyles} />,
```
Comment (karrui) [inline frontend/src/hooks/useMdComponents.tsx]: maybe we can add listStyles here so you don't need the override in `WhatsNewContent`

File: frontend/src/features/whats-new/WhatsNewDrawer.tsx [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+          What’s new
+        </DrawerHeader>
+        <DrawerBody
+          py={0}
+          px={0}
+          mt="1.25rem"
+          display="flex"
+          alignItems="center"
+          flexDirection="column"
+        >
```
Comment (karrui) [inline frontend/src/features/whats-new/WhatsNewDrawer.tsx:67]: do you want to set these styles app wide? Can do in future PR. This is so we can have consistent drawer styles, much like our modal styles

File: frontend/src/features/whats-new/WhatsNewDrawer.tsx [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+          <Stack divider={<StackDivider />} spacing="2rem">
+            {listOfFeatureUpdatesShown.map((featureUpdate, key) => {
+              return (
+                <WhatsNewContent
+                  date={featureUpdate.date}
+                  title={featureUpdate.title}
+                  description={featureUpdate.description}
+                  imageUrl={featureUpdate.imageUrl}
```
Comment (karrui) [inline frontend/src/features/whats-new/WhatsNewDrawer.tsx]: nit: probably can just destructure here

```tsx
<WhatsNewContent {...update} />
```

**Review by karrui — APPROVED** [review]
lgtm otherwise, please request a review from pearly

File: frontend/src/features/whats-new/WhatsNewDrawer.tsx (hunk 2 above)
Comment (siddarth2824) [reply frontend/src/features/whats-new/WhatsNewDrawer.tsx:67]: I will create an issue for this and work on it

**Pull request merged**